.github/workflows/acceptance_tests_reusable.yaml

```diff
@@ -54,7 +54,7 @@ jobs:
           && ( github.event.label.name == 'allow-acceptance-tests' ||  github.event.label.name == 'dependencies' )
       )
     name: "Create pending check"
-    runs-on: ubuntu-latest
+    runs-on: self-hosted
```

Comment:
It seems hosted GHA runners are working now nicely again - Do we really need self-hosted runners right now?

Reply:
When are they going to drop out again ?